cairo-vm-cli/Cargo.toml

```diff
@@ -8,7 +8,7 @@ readme.workspace = true
 keywords.workspace = true
 
 [dependencies]
-bincode = { version = "2.0.0-rc.3", tag = "v2.0.0-rc.3", git = "https://github.com/bincode-org/bincode.git" }
+bincode = { version = "2.0.0-rc.3" }
```

Comment:
Let's use `bincode.workspace = true` instead. That way we ensure no incompatibilities in the future.

Reply:
done